frontend/src/components/notifications/TransferComplete.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div>
+    <span data-test="message">
+      <span v-if="transfer?.transferTime !== undefined"
+        >Transfer completed in {{ transfer?.transferTime }}s.</span
```

Comment:
In my opinion it would be nice to have some additional info about the transfer in the notification. Like from which rollup to which and what token.

Reply:
I also think it doesn't provide much info but we also don't want to bloat them with too much text since we already have all of this info inside the cards. 

I initially thought of linking the toast notification with the transfer card in the following way:
Once the user clicks on the successful toast notification we programatically open the "Activity" tab, scroll to the specific transfer card in the list & expand it.
I gave this suggestion to Olympe but she didn't like it.

Anyways, since this would require a discussion which will lead to a decision, maybe its better to create an issue for this.

Reply:
Ok I also just thought it would be nice to link to the activities tab but if Olympe doesn't like it we shouldn't do it.

Can't we just add the text from the titles in the activities tab? For example "Transferred 1.0 TST to Optimism Goerli Testnet in 5.4 s". Was the text given by Olympe?

Reply:
@GabrielBuragev @manuelwedler I'd much rather a link to the activity tab in the toast, than more info in the toast itself. It could be a link through the underlined "Transferred", for example. 

I don't want the toast itself (as in, the shape) to be clickable because : 1/ it won't be logical that this one is clickable and not the others, 2/ the user would have to click with targeted precision on the toast's shape itself and not the "share on twitter" link.